static/css/footer.less

```diff
@@ -24,121 +24,53 @@
     background: @gray-5;
 
     &_share-icon-list {
-        margin-bottom: unit(30px / @base-font-size-px, em);
+        margin-top: unit(30px / @base-font-size-px, em);
         .list__horizontal();
 
         .respond-to-min(768px, {
-            margin-bottom: 0;
+            margin-top: 0;
+            position: absolute;
+            // Aligns with the baseline of .footer_nav-list text.
+            top: -9px;
+            right: 0;
         });
     }
 
     &_nav-list {
         .list__links();
 
-        .respond-to-max(767px, {
-            border-top-color: @black;
-        });
-
         .list_link {
             font-size: unit(18px / @base-font-size-px, em);
             .u-link__colors(@black);
 
             .respond-to-min(768px, {
-                margin-right: unit(30px / 22px, em);
-                font-size: unit(22px / @base-font-size-px, em);
+                margin-right: unit(18px / @base-font-size-px, em);
```

Comment:
If this is a standard width (30px) gutter with an optical offset to make it look right, consider
`(@grid_gutter-width - 12px)` instead.